feature/secure payments (#2246)
mercadopago/devsite-docs: guides/secure_payments/requirements.en.md

# Prerequisites

| Prerequisites | Description |
|---|---|

> | Mercado Pago Account | It is necessary for the integrator to have a Mercado Pago account in order to generate their credentials. If you don't have one, [click here](https://www.mercadopago[FAKER][URL][DOMAIN]/hub/registration/landing) to create one. |

Reviewer: ```suggestion
| Mercado Pago Account | It is necessary for you to have a Mercado Pago account in order to generate your credentials. If you don't have one, [click here](https://www.mercadopago[FAKER][URL][DOMAIN]/hub/registration/landing) to create one. |
```
